packages/bun-types/ambient.d.ts

```diff
@@ -23,3 +23,9 @@ declare module "*.html" {
   var contents: any;
   export = contents;
 }
+
+declare module "*.svg" {
+  // Bun 1.2.3 added support for frontend dev server
+  var contents: `${string}.svg`;
+  export = contents;
+}
```

Comment:
I think for this one it should create a .d.ts file with it for bun init, as otherwise you will have to add every common extension